pkg/interactive/helper.go

```diff
@@ -0,0 +1,100 @@
+package interactive
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/spf13/cobra"
+)
+
+func GetOidcConfigID(r *rosa.Runtime, cmd *cobra.Command) string {
+	oidcConfigs, err := r.OCMClient.ListOidcConfigs()
+	if err != nil {
+		r.Reporter.Warnf("There was a problem retrieving OIDC Configurations "+
+			"for your organization: %v", err)
+		return ""
+	}
+	if len(oidcConfigs) == 0 {
+		return ""
+	}
+	oidcConfigsIds := []string{}
+	for _, oidcConfig := range oidcConfigs {
+		oidcConfigsIds = append(oidcConfigsIds, fmt.Sprintf("%s | %s", oidcConfig.ID(), oidcConfig.IssuerUrl()))
+	}
+	oidcConfigId, err := GetOption(Input{
+		Question: "OIDC Configuration ID",
+		Help:     cmd.Flags().Lookup("oidc-config-id").Usage,
+		Options:  oidcConfigsIds,
+		Default:  oidcConfigsIds[0],
+		Required: true,
+	})
+	if err != nil {
+		r.Reporter.Errorf("Expected a valid OIDC Config ID: %s", err)
+		os.Exit(1)
+	}
+	return strings.TrimSpace(strings.Split(oidcConfigId, "|")[0])
+}
+
+func GetInstallerRoleArn(r *rosa.Runtime, cmd *cobra.Command,
```

Comment:
The code is similar to lines `811-860` in `cmd/create/cluster/cmd.go`, can we please reuse the function in `create cluster` as well?

So it will be easier to maintain, and logic won't diverge in the future.

Reply:
I didn't do this one as I have plans to change the one in cluster creation to unify the set of roles by the prefix

Reply:
Would it be the same for this flow?
When getting the installer role, select an installer role that is part of a complete set.
Otherwise, the user can select an installer role from an incomplete set.

We have to take into consideration also Hypershift roles, as the set is going to include only three roles.

Reply:
Anyhow, I assume we can refactor this in a sequence PR.